### ISSUE TITLE
Rich Text Display but in revisions

### ISSUE THREAD

**Issue by fritosxii** [matrix-msu/Kora3#508]
Same fix as before: #499 

But on the revisions page

**Comment by fritosxii** [comment]
![screen shot 2019-02-25 at 3 29 31 pm](https://user-images.githubusercontent.com/4040076/53366997-f8ed3300-3912-11e9-9a6c-2aef82270b4e.png)

**Comment by suuuth** [comment]
Branch: `suth-508`

**Comment by suuuth** [comment]
This may not be a front-end issue.  Somehow the page structure is all off.  It should be
```
|-section.revisions
|--revision_card
|--revision_card
|--revision_card
|--revision_card
```

But on the page in the image its:
```
|-section.revisions
|--revision_card
|--revision_card
|---revision_card
|----revision_card
|-----revision_card
|-----revision_card
|----revision_card
|----revision_card
|---revision_card
|---revision_card
|-revision_card
|-revision_card
```
Where the bottom two `|-revision_card`s are what is appearing on the left side of the page in the screenshot.  I'll keep rooting around for a cause

**Comment by suuuth** [comment]
Well this page suddenly started appearing closer to what it should look like.  There is still revision cards nested within revision cards, but the horrible full-height cards in the screenshot are no longer there.  Not sure what changed.  The issue here is just that the content of the richtext field is being returned as HTML, rather than as a string that happens to contain HTML.  This is only true for enslaved.kora3, and not for kora3.

**Comment by suuuth** [comment]
Applied fix from #499 to the revisions page.

**Comment by austintruchan** [comment]
I am not seeing the initial issue. But Anthony should test and double check to be sure.

**Comment by austintruchan** [comment]
Wait. I am seeing this error when trying to view revisions on a record that contains rich text. Not sure if its related to this issue or not. /Users/austin.truchan/Desktop/Screen Shot 2019-03-05 at 10.48.12 AM.png

**Comment by austintruchan** [comment]
Github is broken and won't let me edit that last comment to fix the image. That or the image isn't showing because GitHub is broken.

**Comment by austintruchan** [comment]
Ask me for image if interested, otherwise I will return to this issue later to reupload.

**Comment by suuuth** [comment]
Yes if you can please send it to me directly or upload it to imgur and shoot me the link that would be much appreciated

**Comment by austintruchan** [comment]
Kevin says that the issue in my image should be fixed in a previous issue. So I will pass this to code review.

**Comment by suuuth** [comment]
Yes it will be fixed with #488
